library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```diff
@@ -204,11 +204,8 @@
 
 .portal-main-container {
   grid-area: mainContainer;
-  display: grid;
-  grid-template-areas: 
-      "main"
-      "footer";
-  grid-template-rows: 1fr auto;
+  display: flex;
+  flex-direction: column;
   align-content: space-between;
```

Comment:
The flex-wrap: nowrap property prevents align-content from having an effect.

Try setting flex-wrap to something other than nowrap.